feature-group-definitions/background-gradients.yml

```
@@ -0,0 +1,46 @@
+spec: https://drafts.csswg.org/css-images-4/#gradients
+caniuse:
+  - css-gradients
+  - css-repeating-gradients
+  - css-conic-gradients
+compat_features:
+  - css.properties.background-image.gradients
+  # Linear gradients
+  - css.types.image.gradient.linear-gradient
+  - css.types.image.gradient.linear-gradient.doubleposition
+  # - css.types.image.gradient.linear-gradient.hue_interpolation_method
+  # - css.types.image.gradient.linear-gradient.interpolation_color_space
+  - css.types.image.gradient.linear-gradient.interpolation_hints
+  - css.types.image.gradient.linear-gradient.to
+  - css.types.image.gradient.linear-gradient.unitless_0_angle
+  # Radial gradients
+  - css.types.image.gradient.radial-gradient
+  - css.types.image.gradient.radial-gradient.at
+  - css.types.image.gradient.radial-gradient.doubleposition
+  # - css.types.image.gradient.radial-gradient.hue_interpolation_method
+  # - css.types.image.gradient.radial-gradient.interpolation_color_space
+  - css.types.image.gradient.radial-gradient.interpolation_hints
+  # Conic gradients
+  # - css.types.image.gradient.conic-gradient
+  # - css.types.image.gradient.conic-gradient.doubleposition
+  - css.types.image.gradient.conic-gradient.hue_interpolation_method
+  - css.types.image.gradient.conic-gradient.interpolation_color_space
```

Why not include css.types.image.gradient.conic-gradient but include some of its subentries? The precise opposite choices are more similar to linear and radial gradients above:

```suggestion
  # Conic gradients
  - css.types.image.gradient.conic-gradient
  - css.types.image.gradient.conic-gradient.doubleposition
  # - css.types.image.gradient.conic-gradient.hue_interpolation_method
  # - css.types.image.gradient.conic-gradient.interpolation_color_space
```

---

Yes, this is the right thing to do.